### PR TITLE
Multiple Options API

### PR DESCRIPTION
**Pull request by Maliek**
- [x] tests
- [x] moving to separate script
- [x] create new options on demand
- [x] go over all options until empty is found (while loop)
- [x] switch `name` to `title` in Poll ( ⚠️ you'll need to update your database)
- [x] more tests (multiple options and let him throw da error)

### CODE REVIEW

**Comment by Haroenv** [comment]
ok cava veel suckses

**Comment by Haroenv** [comment]
Did you find a library that serialises the form that way?

File: src/main/java/com/pollistics/controllers/PollController.java [hunk 1]
```diff
@@ -40,16 +40,17 @@ public String pollDetail(@PathVariable String pollId, Model model, HttpServletRe
...
 		String title = request.getParameter("title");
-		String option1 = request.getParameter("option1");
-		String option2 = request.getParameter("option2");
-		String option3 = request.getParameter("option3");
 		HashMap<String, Integer> options = new HashMap<>();
-		options.put(option1, 0);
-		options.put(option2, 0);
-		options.put(option3, 0);
+		int i = 0;
+		while (request.getParameter("option" + i) != "") {
```
Comment (elsmr) [inline src/main/java/com/pollistics/controllers/PollController.java]: `!request.getParameter("option" + i).trim().isEmpty();`

**Comment by Haroenv** [comment]
I'm under impression Maliek!

**Review by Haroenv — APPROVED** [review]
Very good! Maybe a check for multiple options too?

**Comment by Haroenv** [comment]
@eliasmeire I think you reverted some change in application tests, it's supposed to be searching for `Pollistics`, and not `Amerikaanse verkiezingen`

File: src/integTest/java/com/pollistics/PollisticsApplicationTests.java [hunk 2]
```diff
@@ -45,7 +45,7 @@ public void pollByIdTest() {
 
 		ResponseEntity<String> response = restTemplate.getForEntity("/polls/" + poll.getId(), String.class);
 		assertEquals(response.getStatusCodeValue(), 200);
-		assertThat(response.getBody(), containsString(poll.getName()));
+		assertThat(response.getBody(), containsString(poll.getTitle()));
```
Comment (Haroenv) [inline src/integTest/java/com/pollistics/PollisticsApplicationTests.java:48]: This causes the test to fail I think

Comment (Haroenv) [reply src/integTest/java/com/pollistics/PollisticsApplicationTests.java:48]: fixed in 2228110 I think @eliasmeire

**Pull request merged**